flow/connectors/bigquery/bigquery.go

```diff
@@ -295,10 +295,14 @@ func (c *BigQueryConnector) ReplayTableSchemaDeltas(
 
 		for _, addedColumn := range schemaDelta.AddedColumns {
 			dstDatasetTable, _ := c.convertToDatasetTable(schemaDelta.DstTableName)
+			addedColumnBigQueryType := qValueKindToBigQueryType(addedColumn.ColumnType)
```

Comment:
qValueKindToBigQueryType should be fixed to handle this instead

Reply:
In SetupNormalize it needs the exact `FieldType` so `"FLOAT64"` won't work there:
```Golang
columns = append(columns, &bigquery.FieldSchema{
    Name:     column.Name,
    Type:     qValueKindToBigQueryType(genericColType),
    Repeated: qvalue.QValueKind(genericColType).IsArray(),
})
```

Reply:
I've moved this to a new function `qValueKindToBigQueryTypeString`